pkg/actions/id.go

```diff
@@ -0,0 +1,33 @@
+package actions
+
+import (
+	"time"
+
+	"github.com/rs/xid"
+
+	gonanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/treeverse/lakefs/pkg/graveler"
+)
+
+type IDGenerator interface {
```

Comment:
Do we want to move the NewHookRunID functionality also under this interface?

Reply:
I think not because it's the same for both implementations.